antenna-documentation/src/site/markdown/analyzers/ort-result-analyzer-step.md

```diff
@@ -0,0 +1,27 @@
+## ORT Result Analyzer
+This analyzer investigates a given *yaml* file that was created by the [*OSS Review Toolkit* (ORT)](https://github.com/heremaps/oss-review-toolkit/) from *heremaps*.
+
+This analyzer collects the dependencies of a given result of a project that was scanned or analyzed with ORT and has the corresponding yaml output format.  
+ORT has several possible steps that can be performed. 
+This analyzer only supports results created from the *Scan* and *Analyze* steps of the tool.
+
+For more information on how to use and execute the *OSS Review Toolkit* we refer you to their [repository](https://github.com/heremaps/oss-review-toolkit/).
+
+### How to use
+Add the following step into the `<analyzers>` section of your workflow.xml
+
+```xml 
+        <step>
+                    <name>Ort Result Analyzer</name>
+                    <classHint>org.eclipse.sw360.antenna.ort.workflow.analyzers.OrtResultAnalyzer</classHint>
+                    <configuration>
+                        <entry key="base.dir" value="${project.basedir}"/>
+                        <entry key="file.path" value="ort/scan-result.yml"/>
+                    </configuration>
+        </step>
+```
+
+#### Explanation of parameters
+* `base.dir`: The REST API URL of your SW360 instance.
```

Comment:
```suggestion
* `base.dir`: the base dir
```

Reply:
Shouldn't the basedir already be known to the antenna execution? Maybe somewhere in the config? Why would someone configure that path?

Reply:
I added the base.dir again, because the ORTResultAnaylzer extends the ManualAnalyzer which requires the base.dir entry in its configuration